database/migrations/2022_11_09_013847_create_mentor_training_times_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
I'm not sure that this migration is necessary - TrainingController INS/MTR scheduling functionality is no longer used (was replaced by Setmore). #202 is a proposal to remove this from the codebase.

Reply:
The only issue is if we add a migration to remove this and this migration doesn't exist, everything will be fine on prod but anywhere else the migrations will fail and we won't be able to migrate again without back-filling this migration.

Reply:
Ok, sounds reasonable to me